logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/selfDeletingMessages/ObserveSelfDeletionTimerSettingsForConversationUseCase.kt

```diff
@@ -66,12 +65,12 @@ class ObserveSelfDeletionTimerSettingsForConversationUseCaseImpl internal constr
 
     private fun onTeamEnabled(conversation: Either<StorageFailure, Conversation>, considerSelfUserSettings: Boolean): SelfDeletionTimer =
         conversation.fold({
-            SelfDeletionTimer.Enabled(ZERO)
+            SelfDeletionTimer.Enabled(null)
```

Comment:
The idea is replacing zero duration with null for enabled? Looks like an improvement to me. Better have the compiler remind us that it may or may not exist, then having to actively think about the possibility of it being zero.

Reply:
Exactly, it was non-nullable value but then every timer we use timer we check if it is ZERO and map it again to null 🤔